themes/vue/source/css/_common.styl

```diff
@@ -57,9 +57,21 @@ a.button
   transition: all .15s ease
   box-sizing: border-box
   border: 1px solid lighten($green, 8%)
+  &.icon
+    position: relative
+    text-indent: 1.4em;
+    > svg
+      position: absolute
+      left: 0.4em
+      top: 0.4em
+      width: 2em
```

Comment:
this might be too broad of a style, it would be better, IMO, to have a more unique class, to avoid collisions.